src/parser/config/nodeTypesConfig.json

```diff
@@ -337,7 +337,7 @@
     },
     {
         "expression": "is_pattern_expression",
-        "metrics": ["mcc"],
```

Comment:
This fixes that the code counted the pattern match (x is true) as a logical expression, but that should only be the case for and and or patterns.